packages/style/src/config/tokens/core/_core-typography.scss

```diff
@@ -10,12 +10,15 @@ $title-font-family: $font-family-english-heading, $font-family-other-languages;
 
 // Font definition shorthands per semantics atom definitions
 $font-h1-bold: 700 32px / 40px var(--title-font-family);
+$font-h1-medium: 600 32px / 40px var(--title-font-family);
 $font-h1-normal: 500 32px / 40px var(--title-font-family);
 $font-h1-light: 300 32px / 40px var(--title-font-family);
 $font-h2-bold: 700 24px / 30px var(--title-font-family);
+$font-h2-medium: 600 24px / 30px var(--title-font-family);
 $font-h2-normal: 500 24px / 30px var(--title-font-family);
 $font-h2-light: 300 24px / 30px var(--title-font-family);
-$font-h3-bold: 600 18px / 24px var(--title-font-family);
+$font-h3-bold: 700 18px / 24px var(--title-font-family);
```

Comment:
I know it's a really minor change but are we ok with that?

Reply:
I think we are :)